Engine/Source/Runtime/Rendering/WorldRenderer.cpp

```diff
@@ -114,13 +112,13 @@ void WorldRenderer::Render(float deltaTime)
 	const cd::Transform& cameraTransform = m_pCurrentSceneWorld->GetTransformComponent(m_pCurrentSceneWorld->GetMainCameraEntity())->GetTransform();
 	SkyComponent* pSkyComponent = m_pCurrentSceneWorld->GetSkyComponent(m_pCurrentSceneWorld->GetSkyEntity());
 
-	auto lightEntities = m_pCurrentSceneWorld->GetLightEntities();
+	const auto lightEntities = m_pCurrentSceneWorld->GetLightEntities();
 	size_t lightEntityCount = lightEntities.size();
 
 	// Blit RTV to SRV to update light shadow map
 	for (int i = 0; i < lightEntityCount; i++)
 	{
-		auto lightComponent = m_pCurrentSceneWorld->GetLightComponent(lightEntities[i]);
+		const auto lightComponent = m_pCurrentSceneWorld->GetLightComponent(lightEntities[i]);
```

Comment:
```suggestion
		const auto* lightComponent = m_pCurrentSceneWorld->GetLightComponent(lightEntities[i]);
```
Add `*` or `&` to `auto` is more safe.